src/components/sidebar/components/sidebar-nav-list/index.tsx

```diff
@@ -4,8 +4,17 @@
  */
 
 import { ReactNode } from 'react'
+import classNames from 'classnames'
 import s from '../../sidebar.module.css'
 
-export default function SidebarNavList({ children }: { children: ReactNode }) {
-	return <ul className={s.navList}>{children}</ul>
+type NativeDivElement = JSX.IntrinsicElements['div']
+
+export default function SidebarNavList({
+	className,
+	children,
+}: {
+	className?: NativeDivElement['className']
+	children: ReactNode
+}) {
```

Comment:
What do you think about changing it to this? That way it can be more flexible with the props passed to it
```suggestion
}: NativeDivElement & { children: ReactNode }) {
```

Reply:
That's a good strategy 👍 Committed!